tests/integration.suite.yml

```diff
@@ -0,0 +1,9 @@
+# Codeception Test Suite Configuration
+#
+# Suite for unit or integration tests that require WordPress functions and classes.
+
+actor: WpunitTester
```

Comment:
Does it matter if the actor name doesn't match the suite name?

Reply:
Not really, as long as the name is referenced, I see the value of coherence, will try to update across the board to eliminate possible confusion.